pkg/model/api.go

```diff
@@ -51,6 +56,97 @@ func ImportModel(ctx context.Context, filePath, projectId, message, modelType, b
 		return err
 	}
 
-	log.Println("Starting import model job. JobId: ", importModelData.GetImportModelJobId())
+	importModelJobId := importModelData.GetImportModelJobId()
+	if waitForResults {
+		okStatus, job, err := waitForImportModelJob(ctx, importModelJobId)
+		if err != nil {
+			return err
+		}
+		if okStatus {
+			log.Println("Successfully imported model")
+			if mappingYaml != "" {
+				printMappingValidationErrors(ctx, job.VersionId, projectId, mappingYaml)
+			}
+		} else {
+			log.Println("Failed to import model")
```

Comment:
and you need to return an error here (why usilay not return error and not printing them unless we need some special error priniting)

Reply:
Updated